### PR TITLE
docs(api): add console sm admin api

### PR DESCRIPTION
**Pull request by urbanisierung**
## Description

- Relates https://github.com/camunda-cloud/camunda-cloud-management-apps/issues/2871 https://github.com/camunda/product-hub/issues/2121
- Uses interactive api explorer

image

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by urbanisierung** [comment]
@pepopowitz iirc you've introduced the interactive API explorer to the docs. Can you please have a quick look if anything is missing in this PR? Thanks!

File: api/console-sm/console-sm-openapi.yaml
Comment (pepopowitz) [inline api/console-sm/console-sm-openapi.yaml:1]: non-blocking recommendation: 

Descriptions on requests, responses, and fields provide helpful guidance in the generated API docs. A good illustration is to compare the docs generated from this spec (on the left) to the [Tasklist REST API docs](https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/specifications/search-tasks/) (on the right): 

image


It's very likely that you're all thinking about this already, but I just wanted to provide another nudge to fill descriptions in. I have work lined up in the somewhat-near-future to follow up with OpenAPI specs that are lacking this data, and open PRs at the source; it's not my immediate priority, and if you accomplish that before I do that's great.

**Review by pepopowitz — CHANGES_REQUESTED** [review]
This is really fantastic!!!

I have a couple small blocking requests, and one non-blocking. Overall, it's great, and I want to thank you for following the patterns in place and practically nailing this implementation. It looks really good.

I'm also curious if you have feedback for me on what it takes to add a new API Explorer. Were there challenges along the way that seemed frustrating or unnecessary? Any opportunities for improvement you noticed?

File: docs/apis-tools/console-sm-api/specifications/console-sm-admin-api.info.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import ApiLogo from "@theme/ApiLogo";
+import SchemaTabs from "@theme/SchemaTabs";
+import TabItem from "@theme/TabItem";
+import Export from "@theme/ApiExplorer/Export";
+
+<span className={"theme-doc-version-badge badge badge--secondary"}>
+  Version: 1.0.0
+</span>
+
```
[the remark this answers is not in the document]
Comment (urbanisierung) [reply docs/apis-tools/console-sm-api/specifications/console-sm-admin-api.info.mdx]: Got it. By fixing the filename it was removed by the script.

File: api/console-sm/console-sm-openapi.yaml
Comment (urbanisierung) [reply api/console-sm/console-sm-openapi.yaml:1]: @pepopowitz Thanks for the hints! I've updated our OpenAPI spec re descriptions:

image

**Comment by urbanisierung** [comment]
> I'm also curious if you have feedback for me on what it takes to add a new API Explorer. Were there challenges along the way that seemed frustrating or unnecessary? Any opportunities for improvement you noticed?

@pepopowitz I used a similar docusaurus plugin (maybe the same?) for one of my side project, so I already had same background knowledge what files to touch.

As a general improvement from my point of view:

* Our OpenAPI lib creates a json file. haven't tried the plugin with a `json` as input. If it's not supported it would be nice to have a built-in way within the docs repo to use also `json`, for example with a json-to-yaml converter (also available as npm module)
* What would be super cool: auto-fetching of OpenAPI spec instead of manually copying it. Our OpenAPI specs are (semi-) publicly available (via github or public url). The script could automatically fetch the latest content and update the docs.

File: api/console-sm/console-sm-openapi.yaml
Comment (pepopowitz) [reply api/console-sm/console-sm-openapi.yaml:1]: Amazing, thank you!!!

**Review by pepopowitz — APPROVED** [review]
❤️ ❤️ ❤️ ❤️

**Comment by christinaausley** [comment]
Hi @urbanisierung 😄 Is this still a WIP, or are you planning to make additional changes and/or looking for additional review?

**Comment by urbanisierung** [comment]
> Hi @urbanisierung 😄 Is this still a WIP, or are you planning to make additional changes and/or looking for additional review?

@christinaausley Not planning to make any additional changes. Unfortunately I'm waiting for green lights from QA for this as well.

**Comment by urbanisierung** [comment]
@christinaausley We finally got green lights from QA. The changes will be part of 8.6 alpha4 - can I just merge this one?

**Comment by christinaausley** [comment]
@urbanisierung FYI that I'll get this into Postman for release next week.

**Pull request merged**